### ISSUE TITLE
Fix letter_opner in production

### ISSUE THREAD

**Issue by n350071** [n350071/eventer#30]
```
-----> Installing node-v10.15.3-linux-x64
-----> Installing yarn-v1.16.0
-----> Detecting rake tasks
-----> Preparing app for Rails asset pipeline
       Running: rake assets:precompile
       rake aborted!
       NameError: uninitialized constant LetterOpener
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `block in load_missing_constant'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:8:in `without_bootsnap_cache'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `rescue in load_missing_constant'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:58:in `load_missing_constant'
       /tmp/build_49807e919be2126095d2e4433206295a/config/initializers/letter_opener.rb:2:in `<main>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:667:in `block in load_config_initializer'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.0/lib/active_support/notifications.rb:182:in `instrument'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:666:in `load_config_initializer'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:624:in `block (2 levels) in <class:Engine>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:623:in `each'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:623:in `block in <class:Engine>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:32:in `instance_exec'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:32:in `run'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:61:in `block in run_initializers'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:50:in `each'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:50:in `tsort_each_child'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:60:in `run_initializers'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/application.rb:363:in `initialize!'
       /tmp/build_49807e919be2126095d2e4433206295a/config/environment.rb:5:in `<main>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/zeitwerk-2.2.1/lib/zeitwerk/kernel.rb:23:in `require'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/application.rb:339:in `require_environment!'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/application.rb:515:in `block in run_tasks_blocks'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/sprockets-rails-3.2.1/lib/sprockets/rails/task.rb:62:in `block (2 levels) in define'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/rake-13.0.0/exe/rake:27:in `<top (required)>'
       
       Caused by:
       NameError: uninitialized constant LetterOpener
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:60:in `block in load_missing_constant'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:16:in `allow_bootsnap_retry'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:59:in `load_missing_constant'
       /tmp/build_49807e919be2126095d2e4433206295a/config/initializers/letter_opener.rb:2:in `<main>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:54:in `load'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:667:in `block in load_config_initializer'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.0/lib/active_support/notifications.rb:182:in `instrument'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:666:in `load_config_initializer'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:624:in `block (2 levels) in <class:Engine>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:623:in `each'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/engine.rb:623:in `block in <class:Engine>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:32:in `instance_exec'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:32:in `run'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:61:in `block in run_initializers'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:50:in `each'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:50:in `tsort_each_child'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/initializable.rb:60:in `run_initializers'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/application.rb:363:in `initialize!'
       /tmp/build_49807e919be2126095d2e4433206295a/config/environment.rb:5:in `<main>'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/zeitwerk-2.2.1/lib/zeitwerk/kernel.rb:23:in `require'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/application.rb:339:in `require_environment!'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/railties-6.0.0/lib/rails/application.rb:515:in `block in run_tasks_blocks'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/sprockets-rails-3.2.1/lib/sprockets/rails/task.rb:62:in `block (2 levels) in define'
       /tmp/build_49807e919be2126095d2e4433206295a/vendor/bundle/ruby/2.6.0/gems/rake-13.0.0/exe/rake:27:in `<top (required)>'
       Tasks: TOP => environment
       (See full trace by running task with --trace)
 !
 !     Precompiling assets failed.
 !
 !     Push rejected, failed to compile Ruby app.
 !     Push failed
```

**Comment by n350071** [comment]
treatment

```ruby
if Rails.env.development?
  # https://github.com/ryanb/letter_opener#configuration
  LetterOpener.configure do |config|
    # To overrider the location for message storage.
    # Default value is <tt>tmp/letter_opener</tt>
    config.location = Rails.root.join('tmp', 'my_mails')

    # To render only the message body, without any metadata or extra containers or styling.
    # Default value is <tt>:default</tt> that renders styled message with showing useful metadata.
    config.message_template = :light
  end
end
```

**Comment by n350071** [comment]
commit 94d117d0049eeeb18503dc66dd64ce91a80b9dbc
Author: n350071 <n350071@gmail.com>
Date:   2020-01-08 06:19:06 +0900

    :bug: Fix letter_opner in production #30
